CHANGELOG.md

```diff
@@ -1,8 +1,26 @@
 # Changelog
 
+## 1.0.0b5 / 2021-01-13
+
+* [Added] Add new live and rehydrated logs breakdowns for Usage API to public documentation. See [#223](https://github.com/DataDog/datadog-api-client-python/pull/223).
```

Comment:
Should we remove this if its just docs?

Reply:
Well it's not just docs, title is misleading, I'll update it 😅